### DISCUSSION TITLE
内置的Mosdns好像不支持http server还有quic server

### DISCUSSION THREAD

**Discussion by Vincent-bin** [kkkgo/PaoPaoDNS#79] [想法]
看了一下脚本好像不是用的官方的那个mosdns

**Comment by kkkgo** [comment]
是的，本项目的mosdns非官方版本mosdns，功能根据实际需求有增有删，配置文件也不完全兼容，在说明文档中没有描述的功能一般不会被保留以优化性能和体积。项目链接包含在附录中。`mosdns.yaml`仅供调试使用而非自定义功能，镜像升级该文件会被覆盖。

`http/quic server`的应用场景非常有限。因为运营商和地理位置的差异，递归DNS一般更适合在局域网中使用。而境外的场景，FAKEIP是效果更好的方案。如果需要`http/quic server`，建议套一个`AdGuard Home`，不仅支持`http/quic server`，还具备Web的操作界面和查询日志，搭建和维护都更为轻松。

**Reply by Vincent-bin** [reply]
好的明白了
